recipe/build.sh

```diff
@@ -5,10 +5,7 @@ if [ "$NEED_SCRIPTS" == no ]; then
 fi
 rm ${SP_DIR}/anaconda_anon_usage/plugin.py
 mkdir -p "${PREFIX}/etc/conda/activate.d"
-mkdir -p "${PREFIX}/python-scripts"
 cp "scripts/activate.sh" "${PREFIX}/etc/conda/activate.d/${PKG_NAME}_activate.sh"
-cp "scripts/activate.bat" "${PREFIX}/etc/conda/activate.d/${PKG_NAME}_activate.bat"
-cp "scripts/post-link.sh" "${PREFIX}/python-scripts/.${PKG_NAME}-post-link.sh"
-cp "scripts/post-link.bat" "${PREFIX}/python-scripts/.${PKG_NAME}-post-link.bat"
-cp "scripts/pre-unlink.sh" "${PREFIX}/python-scripts/.${PKG_NAME}-pre-unlink.sh"
-cp "scripts/pre-unlink.bat" "${PREFIX}/python-scripts/.${PKG_NAME}-pre-unlink.bat"
+mkdir -p "${PREFIX}/bin"
+cp "scripts/post-link.sh" "${PREFIX}/bin/.${PKG_NAME}-post-link.sh"
+cp "scripts/pre-unlink.sh" "${PREFIX}/bin/.${PKG_NAME}-pre-unlink.sh"
```

Comment:
Removing mention of `.bat` and `python-scripts` here because these are arch-specific builds.